### PR TITLE
Fix for get_unique_id method

### PR DESCRIPTION
**Pull request by sebworks**
Fixing get_unique_id method to return hex based time stamp

## Additions

- Fixing get_unique_id method to return hex based time stamp

## Testing

- View filter checkbox ids on the `http://localhost:8000/the-bureau/leadership-calendar/` page.

## Review
- @anselmbradford 
- @KimberlyMunoz 
- @kave

### CODE REVIEW

File: cfgov/v1/util/util.py [hunk 1]
```diff
@@ -15,14 +17,20 @@ def to_camel_case(snake_str):
     components = snake_str.split('_')
     return components[0] + "".join(x.title() for x in components[1:])
 
+
+def get_unique_id(prefix='', suffix=''):
+    index = hex(int(time()*10000000))[2:]
```
Comment (anselmbradford) [inline cfgov/v1/util/util.py]: Why does time need to be multiplied by `10000000`?

Comment (sebworks) [inline cfgov/v1/util/util.py]: It's a floating point number so you want to include the milliseconds before casting to integer. That is what will help make it unique if you have concurrent calls to `get_unique_id`.

Comment (anselmbradford) [inline cfgov/v1/util/util.py]: Why 10 million though? Isn't it a float out only two decimal places?

Comment (sebworks) [inline cfgov/v1/util/util.py]: So I think it depends on float point precision.  `time.time()` returns `1451584127.121017` at this moment.

Comment (anselmbradford) [inline cfgov/v1/util/util.py]: Hm, interesting. I set `index = time()` and it was two spaces in the output HTML for me. :+1:

**Comment by anselmbradford** [comment]
:+1:

File: cfgov/v1/util/util.py (hunk 1 above)
Comment (sebworks) [inline cfgov/v1/util/util.py]: Yep, same for me when the floating point number is converted into a string. 

`str(time.time())` produces `1451585483.05`

**Pull request merged**